packages/x-charts/src/context/SizeProvider/useChartContainerDimensions.ts

```diff
@@ -48,9 +46,8 @@ export const useChartContainerDimensions = (
     // computeRun is used to avoid infinite loops.
     if (
       hasInSize ||
-      !resolveSizeBeforeRender ||
       !stateRef.current.initialCompute ||
```

Comment:
If you don't need the `resolveSizeBeforeRender` you can remove it from the props. The PR change migth impact much more files with the scripts ;)

Reply:
yep, had the commit ready, but there would be a lot of file changes

Do you think we need a codemod? I don't think a lot of people would be using it 🙃

Reply:
Added a BC to changelog/migration docs